pkgpanda/cli.py

```diff
@@ -166,6 +167,10 @@ def main():
                 print(pkg)
             sys.exit(0)
 
+        if arguments['show']:
+            actions.show_package(repository, arguments['<package-id>'])
```

Comment:
Per my other inlined comment, we can get away with a simple one-liner here:

```suggestion
            print(json.dumps(repository.load(arguments['<package-id>']).pkginfo, indent=4))
```

However I'm not sure if JSON is the best output format, it's not very human readable compared to e.g . YAML. If you need the JSON, maybe we can default to something more human readable and then support an optional `--json` flag to print JSON instead?

Reply:
I think JSON is fine as output format here, since the underlying `pkginfo.json` was originally json-formatted anyways.

Reply:
I don't think the storage format is relevant to how we display it in the terminal. Pkgpanda CLI subcommands produce human-readable output, and I'd like that to stay consistent. A `--json` flag or similar should be suitable for use cases that need machine-readable output.

Reply:
I'm not sure that I follow your reasoning, can you explain why you think that JSON is not human-readable, and that YAML would be human-readable?

Reply:
> I'm not sure that I follow your reasoning, can you explain why you think that JSON is not human-readable, and that YAML would be human-readable?

YAML's purpose is to be a "human-friendly" serialization format, with syntax that's intended to be friendly to the eye. https://en.wikipedia.org/wiki/YAML I tossed it out as an example since it better resembles the output of the other pkgpanda subcommands, but I'm open to others, or even just a `for` loop of `print` statements. I'm concerned with keeping this CLI consistent as new subcommands are added.